ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -8,7 +10,34 @@ const triggerSet: TriggerSet<Data> = {
   id: 'AacCruiserweightM2Savage',
   zoneId: ZoneId.AacCruiserweightM2Savage,
   timelineFile: 'r6s.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R6S Mousse Mural',
+      type: 'StartsUsing',
+      netRegex: { id: 'A6BC', source: 'Sugar Riot', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R6S Color Clash',
+      type: 'StartsUsing',
+      netRegex: { id: ['A68D', 'A68B'], source: 'Sugar Riot' },
+      alertText: (_data, matches, output) => {
+        if (matches.id === 'A68D')
+          return output.stackPartner!();
+        return output.healerGroups!();
+      },
+      outputStrings: {
+        stackPartner: Outputs.stackPartner,
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'R6S Sticky Mousse',
+      type: 'StartsUsing',
+      netRegex: { id: 'A695', source: 'Sugar Riot', capture: false },
+      response: Responses.protean(),
```

Comment:
```suggestion
      response: Responses.spread(),
```
Protean meanly refers to bait cone/line AOE around target in 8 directions. This mech is more like a spread. 
There are also strat that stand 2 lines instead of 8 directions. Meanwhile, a double stack is right after it, but spreadThenStack response not include the double stack, so it seems better to custom an output.